moveit_setup_assistant/launch/setup_assistant.launch.py

```diff
@@ -0,0 +1,14 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+    # moveit_setup_assistant executable
+    moveit_setup_assistant = Node(
+        name="moveit_setup_assistant",
+        package="moveit_setup_assistant",
+        executable="moveit_setup_assistant",
+        output="screen",
+    )
+
+    return LaunchDescription([moveit_setup_assistant])
```

Comment:
Please drop these changes from this PR (and restore the old launch file you deleted).